### DISCUSSION TITLE
選舉快到了很少有政治以外的新聞

### DISCUSSION THREAD

**Discussion by yuchien-lin** [1101-GS4719-C/Python-Community-GS4719-C#1753] [Q&A]
最近因為選舉快到了，很少有政治以外的新聞，像是前幾天發生的中壢家樂福失火，就是我去查新聞的網站才看到的，大家平常都是用什麼去看最近的新聞呢?

**Comment by THH1118** [comment]
我自己是會去FB追蹤特定類別的粉專或社團，裡面分享的就會是那個類別的新聞，那種大的新聞台應該都只會報熱門有看點的新聞吧哈哈

**Comment by WeiYen1103** [comment]
這我真的很有感，現在選舉將近，我感覺各家媒體似乎都認為要播報甚麼藍白合記者會才有流量，才有焦點，但那些真正需要被聽見的聲音呢，我覺得媒體素養真的很是重要阿...

**Comment by uwje** [comment]
感覺選舉才是各政黨跟政治人物的主場，我反而覺得他們認為執政對他們來說才是附屬品，做得好不好其實沒關係，買個媒體開始洗地就好，順便隨手刷幾個對手的黑料就好，選上後就開始斂財，提升政黨實力，等著下次的選舉。

**Comment by davidteng00000** [comment]
每次回家打開電視都是滿滿的政治新聞，我有時候會被立場偏頗的新聞誤導，然後發現被誤導之後又很憤怒，造成我的情緒起起伏伏，所以我乾脆都不看新聞了

**Comment by 1chooo** [comment]
我現在很多的社群也都開始充斥政治相關的新聞，平常不太冒出來的人都跟著出現了，不想看到也很難，總有些言論特別的偏激⋯⋯

**Comment by uwje** [comment]
但選舉快到了，的確會增加非選舉類新聞出現的機率，但我認為不至於到完全看不到相關新聞，畢竟還是會有新聞台播放相關新聞。

**Comment by chuanmin0317** [comment]
的確每次選舉就會有一堆類似的新聞出現，而最近幾年我觀察到，網路新聞的留言區也是烏煙瘴氣，每個人都會批評他們不喜歡的候選人，而且有些帳號還是新註冊的，真的蠻討厭這種環境
